src/schema/v2/artist/meta.ts

```diff
@@ -1,45 +1,91 @@
 import { stripTags, truncate, markdownToText } from "lib/helpers"
 import { compact } from "lodash"
-import { GraphQLString, GraphQLObjectType, GraphQLFieldConfig } from "graphql"
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLFieldConfig,
+  GraphQLEnumType,
+} from "graphql"
 import { ResolverContext } from "types/graphql"
 
 export const metaName = (artist) => {
   if (artist.name) return stripTags(artist.name)
   return "Unnamed Artist"
 }
 
+export const formatDescription = (description: string, blurb?: string) => {
+  return truncate(compact([description, blurb]).join("."), 157)
```

Comment:
I don't think this join is what you want. Though it's not clear to me what we do want here? What types of content are in the description and blurb?

Reply:
we want the bio first sentence after the description as specified in [this doc](https://docs.google.com/document/d/1P7f8LINemT1avr_4xU9sBYroefh5b9h3Gkar3vymtKE/edit) -- i was just recycling what was in there before but you're right! i could just build a new string aka `${description} ${blurb}` and truncate the whole thing. Lmk if you have a smoother way to do dis in mind

Reply:
I mean it's joining with simply a `'.'` (no space). You would want to join with `'. '` The compact & join is preferable in the case of nulls.

But this doc is also specifically asking for the first sentence (which honestly I can see that being pretty weird).